intermediate_source/ddp_tutorial.rst

```diff
@@ -80,28 +80,20 @@ be found in
 
     from torch.nn.parallel import DistributedDataParallel as DDP
 
+    # Distributed package only covers collective communications with Gloo
+    # backend, FileStore and TcpStore on Windows platform.
```

Comment:
nit: trying to make it more explicit

On Windows platform, the torch.distributed package only supports Gloo backend,
FileStore and TcpStore.

Reply:
Resolved